internal/kokoro/synth/synth.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e -x
+
+cd $(dirname $0)/../../..
+
+should_create_pr=
+git checkout -b synth-update || {
+    # If a branch already exists, there's already an existing PR.
+    should_create_pr=1;
```

Comment:
I don't understand this. There is already an existing PR, but we should create one?

Reply:
Other way around.

If there's an existing PR, update it
If not, create one

Will re-phrase.

Reply:
oh, but this code is wrong. heh.

Reply:
PTAL